dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/acl/DefaultTrackerAccessManager.java

```diff
@@ -401,7 +402,7 @@ public List<String> canCreate(
 
   @Override
   public List<String> canUpdate(
-      @Nonnull UserDetails user, Event event, boolean skipOwnershipCheck) {
+      @Nonnull UserDetails user, TrackerEvent event, boolean skipOwnershipCheck) {
```

Comment:
TODO: Handle SingleEvent access check. Maybe as a different method? That also enables simplifying this method if it is for TrackerEvent alone. And a specific one for SingleEvent?